### ISSUE TITLE
report crashed qui widgets

### ISSUE THREAD

**Issue by HW42** [QubesOS/qubes-issues#4669]
If a qui widget crashes (for example because of #4668) the error message is only visible on stderr/`~/.xsession-errors`. There should be a more user friendly way to see/notice such errors (maybe a notification?).

**Comment by ben-grande** [comment]
They should be  systemd unit to be logged and properly reported and easy to find with `systemctl --failed`.

**Comment by HW42** [comment]
Yes there are now a systemd user unit, so the log ends in journal instead and the unit state is updated. But I think back when I reported this I was thinking of less experienced users for whom their usual UI suddenly disappears. Like we already have in other places a pointer where to look would be nice.
